Logic/SpacePark/SpaceParkApi.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using RestSharp;
+using SpaceApi.Models;
+
+namespace Logic.SpacePark
+{
+    public class SpaceParkApi : ISpaceParkApi
+    {
+        public async Task<Parking> GetParkingById(int id) // Return the specfied parking lot
+        {
+            var client = new RestClient("https://localhost/api/");
```

Comment:
```suggestion
Could this be made into a separate method to avoid redundancy?
           var client = new RestClient("https://localhost/api/");
            client.AddDefaultHeader("apikey", "secret1234");
            var request = new RestRequest($"parkings?portname={portName}", DataFormat.Json);
            var response = await client.GetAsync<List<Parking>>(request);
            return response;
```